auslib/web/admin/swagger/api.yaml

```diff
@@ -0,0 +1,423 @@
+swagger: '2.0'
+info:
+  version: '2.28'
+  title: "Balrog's Admin API"
+  termsOfService: "https://wiki.mozilla.org/Balrog"
+  license:
+    name: "Mozilla Public License 2.0"
+    url: "https://www.mozilla.org/en-US/MPL/2.0/"
+  contact:
+    name: "Mozilla Release Engineering"
+    url: "https://github.com/mozilla/balrog"
+    email: "release-engineering@lists.mozilla.org"
+  description: "Mozilla's Update Server. APIs for Balrog" #TODO write in detail using Github Flavored Markdown
+
+schemes:
+  - http
```

Comment:
Does removing http break anything (eg: local development)? If it doesn't, please remove it.